### PR TITLE
kubeadm: increase ut coverage certs/renewal/readwriter

### PR DESCRIPTION
**Pull request by my-git9**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

kubeadm: increase ut coverage certs/renewal/readwriter

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go [hunk 1]
```diff
@@ -199,3 +200,147 @@ func writeTestKubeconfig(t *testing.T, dir, name string, caCert *x509.Certificat
...
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := fileExists(tt.filename); got != tt.want {
+				t.Errorf("fileExists() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestPkiCertificateReadWriterExists(t *testing.T) {
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go:249]:
```suggestion
func TestPKICertificateReadWriterExists(t *testing.T) {
```

File: cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go [hunk 2]
```diff
@@ -199,3 +200,147 @@ func writeTestKubeconfig(t *testing.T, dir, name string, caCert *x509.Certificat
...
+		baseName       string
+		certificateDir string
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   bool
+	}{
+		{
+			name: "cert file exist",
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go:276]:
```suggestion
			name: "cert file exists",
```

File: cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go [hunk 3]
```diff
@@ -199,3 +200,147 @@ func writeTestKubeconfig(t *testing.T, dir, name string, caCert *x509.Certificat
...
+	if err != nil {
+		t.Fatalf("Couldn't create tmpfile: %v", err)
+	}
+	tests := []struct {
+		name               string
+		kubeConfigFilePath string
+		want               bool
+	}{
+		{
+			name:               "file exist",
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go:326]:
```suggestion
			name:               "file exists",
```

Comment (neolit123) [reply cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go:326]: this website can help you:
https://conjugator.reverso.net/conjugation-english-verb-exists.html

you can put an english verb like "exists" and it conjugates it.
it also has link for the "not exists" negative form.

**Review by neolit123 — COMMENTED** [review]
thanks for the PR @my-git9 
added some minor comments.

File: cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go (hunk 3 above)
Comment (my-git9) [reply cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go:326]: I know, thanks

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**